CDS-57577-CDS-54890 (#1436)
harness/developer-hub: docs/continuous-delivery/gitops/harness-git-ops-application-set-tutorial.md

![](./static/harness-git-ops-application-set-tutorial-40.png)

<details>

> <summary>SYNC button</summary>

Reviewer: If this is a procedure the user should do or can do, I don't think it should be hidden in an expandable block.

> When you select **Sync**, Harness will use the ApplicationSet to create the new Harness Applications for the dev and prod clusters.

Reviewer: ```suggestion
When you select **Sync**, Harness uses the ApplicationSet to create the new Harness Applications for the dev and prod clusters.
```

1. In the GitOps Application, select **SYNC**.
   
   ![](./static/harness-git-ops-application-set-tutorial-41.png)

> 2. In the Sync settings, select **Synchronize**. Synchronization will take a minute.

Reviewer: I'm not sure what the image above this is showing me. I also don't see Sync settings in the image, so this step is a bit confusing. Maybe in Step 1 you could give a bit more context or explain what the image is... like 'the xx page appears' or something before the image.

</details>

<details>

> <summary>GitOpsSync execution step</summary>

Reviewer: Same comment. Is this a procedure the user can/should do? If so, it shouldn't be in an expandable.

   You will store the PAT in a [Harness Text Secret](/docs/platform/Secrets/add-use-text-secrets). For details on Secrets Management, go to [Harness Secrets Management Overview](/docs/platform/Secrets/Secrets-Management/harness-secret-manager-overview).
    
    7. Select **Enable API access** and use the same Harness Secret.
    
    For more information, go to [add a GitHub connector](/docs/platform/Connectors/Code-Repositories/add-a-git-hub-connector).
3. Select **Continue**.
4. In **Connect to the provider**, select **Connect through Harness Platform**., and then click **Save and Continue**.
5. When the **Connection Test** in complete, click **Continue**.

```mdx-code-block
</TabItem>
<TabItem value="Bitbucket" label="Bitbucket">
```
1. In **Bitbucket Connector**, select **New Bitbucket Connector**.
2. Enter the following Bitbucket Connector settings:
    1. **Name:** enter **gitops-bitbucket-cloud**.
    2. **URL Type:** select **Repository**.
    3. **Connection Type:** select **HTTP**.
    4. **GitHub Repository URL:** enter the HTTP URL for repo you used for your ApplicationSet, such as `https://bitbucket.org/johnsmith/applicationset.git`.

>     5. **Authentication:** select **Username and Token**. For the Token, you'll need to use a Personal Access Token (PAT) from Bitbucket. If you are logged into Bitbucket, just go to [HTTP access tokens](https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html).

Reviewer 2: Use a Personal Access Token (PAT) from Bitbucket. If you are logged into Bitbucket, go to [HTTP access tokens](https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html). You must provide an account-level app password or token. Repo-level tokens are not supported.

       You must provide an account-level app password or token. Repo-level tokens are not supported.
   

>        You will store the PAT in a [Harness Text Secret](/docs/platform/Secrets/add-use-text-secrets). For details on Secrets Management, go to [Harness Secrets Management Overview](/docs/platform/Secrets/Secrets-Management/harness-secret-manager-overview).

Reviewer 2: ```suggestion
       Store the PAT in a [Harness Text Secret](/docs/platform/Secrets/add-use-text-secrets). For details on Secrets Management, go to [Harness Secrets Management Overview](/docs/platform/Secrets/Secrets-Management/harness-secret-manager-overview).
```

    
    6. Select **Enable API access** and use the same Harness Secret.
    
    For more information, go to [Bitbucket connector settings reference](/docs/platform/Connectors/Code-Repositories/ref-source-repo-provider/bitbucket-connector-settings-reference).
3. Select **Continue**.

> 4. In **Connect to the provider**, select **Connect through Harness Platform**., and then click **Save and Continue**.

Reviewer 2: ```suggestion
4. In **Connect to the provider**, select **Connect through Harness Platform**, and then click **Save and Continue**.
```

> 5. When the **Connection Test** in complete, click **Continue**.

Reviewer 2: ```suggestion
5. After the Connection Test is complete, click **Continue**.
```

1. In **Select Service**, click **New Service**.
2. In **Name**, enter **PR Example**.
3. In **Manifests**, click **Add Release Repo Manifest**.
4. In **Release Repo Store**, select one of the following repositories.  

> ```mdx-code-block

Reviewer: Can the tabs code block be indented?

import Tabs from '@theme/Tabs';   
import TabItem from '@theme/TabItem';
```
```mdx-code-block
<Tabs>
    <TabItem value="GitHub" label="GitHub" default>
```

Now we'll add a Github Connector to tell Harness where to pull the config.json from.

1. In **Github Connector**, select **New Github Connector**.

> 2. Enter the following Github Connector settings:

Reviewer: The settings shouldn't be numbered steps. More of a bullet list.

Reviewer: (I know that's beyond what you're doing. :-) But it's glaring.

Author: @StrawberryPad It is a procedure, shouldn't it be numbered?

>    You will store the PAT in a [Harness Text Secret](/docs/platform/Secrets/add-use-text-secrets). For details on Secrets Management, go to [Harness Secrets Management Overview](/docs/platform/Secrets/Secrets-Management/harness-secret-manager-overview).

Reviewer: Please indent lines 394 (image) and 396 ("You will store the PAT...")
